README.md

```diff
@@ -13,39 +13,119 @@ Artifacts in support of ODH Edge use cases that show integration with Red Hat Ad
+   In the Cluster set popup menu, select the cluster set you created for the near edge clusters.
+   If you use the Import mode of "Enter your server URL and API token for the existing cluster",
+   obtain the address of the server and the token on the other cluster's OpenShift Console 
+   at the top-right corner user menu > Copy login command, for an admin user.
+
+#### Create namespaces for applications
+
+For each of the deployed models-turned-into-application, we will create separate namespace/project on the ACM hub cluster.
+The ACM setup in this repository expects namespaces `tensorflow-housing-app` and `bike-rental-app`.
+
+As a regular user in OpenShift Console, in the local-cluster > Developer menu, Project popup, use the "Create Project" button;
+alternatively, run
+```bash
+oc new-project tensorflow-housing-app
+oc new-project bike-rental-app
+
+```
+
+#### Bind namespaces to the cluster set
+
+The newly created namespaces need to be bound the cluster set with the near edge clusters.
+
+On the ACM hub cluster, in OpenShift Console in the All Clusters menu > Infrastructure > Clusters > Cluster sets tab,
+click the three-dot button at the right of the cluster set, select Edit namespace bindings, and add the namespaces you've created.
+
+This is the last step that needs to be done as an admin user with `cluster-admins` privileges.
+
+#### Deploy the applications to remote clusters
+
+With the remote near edge clusters in the cluster set and the cluster set bound to the application namespaces,
+it is now possible to deploy the applications to remote clusters through
+ACM Channels, Subscriptions, Placements, and Applications.
+
+We will create the objects in the respective namespaces from the `acm/registration/` directory structure.
+
+However, it is important to note that the actual application configuration, stored in `acm/odh-edge/`,
+does not come from our local git repository checkout.
+Instead, [acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml](acm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yamlacm/registration/near-edge/overlays/tensorflow-housing-app/kustomization.yaml)
+and [acm/registration/near-edge/overlays/bike-rental-app/kustomization.yaml](acm/registration/near-edge/overlays/bike-rental-app/kustomization.yaml)
+contain URLs of the git repositories that will become configurations of the ACM Channels,
+and the applications will be deployed from those remote git repositories.
+By default, [github.com/opendatahub-io/ai-edge](https://github.com/opendatahub-io/ai-edge) is configured;
+edit the URLs to match your repositories.
+
+Then run as a regular (non-admin) user
+```bash
+oc apply -k acm/registration/
+```
+
+#### View the application deployment
+
+In the OpenShift Console of the ACM hub cluster in the All Clusters > Applications, filter for Subscriptions.
+You should see `tensorflow-housing-app-application` and `bike-rental-app-application` with Git resource.
```

Comment:
```suggestion
You should see `tensorflow-housing-app-application` and `bike-rental-app-application` with Git resources.
```

Reply:
Applied in 49aadf7f7022a32884be3d91d13200f348ad82ab.